sdk/core/azure-core/src/main/java/com/azure/core/util/polling/PollerFlux.java

```diff
@@ -139,6 +150,68 @@ public PollerFlux(Duration pollInterval,
             true);
     }
 
+    /**
+     * Creates PollerFlux.
+     *
+     * This create method uses a {@link PollingStrategy} to poll the status of a long running operation after the
+     * activation operation is invoked. See {@link PollingStrategy} for more details of known polling strategies
+     * and how to create a custom strategy.
+     *
+     * @param pollInterval the polling interval
+     * @param initialOperation the activation operation to activate (start) the long running operation.
+     *     This operation will be invoked at most once across all subscriptions. This parameter is required.
+     *     If there is no specific activation work to be done then invocation should return Mono.empty(),
+     *     this operation will be called with a new {@link PollingContext}.
+     * @param strategy a known strategy for polling a long running operation in Azure
+     * @param pollResponseType the {@link TypeReference} of the response type from a polling call, or BinaryData if raw
+     *                         response body should be kept. This should match the generic parameter {@link U}.
+     * @param resultType the {@link TypeReference} of the final result object to deserialize into, or BinaryData if raw
+     *                   response body should be kept. This should match the generic parameter {@link U}.
+     * @param <T> The type of poll response value.
+     * @param <U> The type of the final result of long running operation.
+     * @return PollerFlux
+     */
+    @SuppressWarnings("unchecked")
+    public static <T, U> PollerFlux<T, U>
+        create(Duration pollInterval,
+               Supplier<Mono<? extends Response<?>>> initialOperation,
+               PollingStrategy<T, U> strategy,
+               TypeReference<T> pollResponseType,
+               TypeReference<U> resultType) {
+        return create(
+            pollInterval,
+            ctx -> initialOperation.get()
+                .flatMap(r -> strategy.canPoll(r).flatMap(canPoll -> {
```

Comment:
nit, might better spell the complete name for `r`. It appears lots of places after, somehow hard to read `r.getValue` (what's getValue?).

Reply:
Renamed to `response`.